plugin/trino-kudu/pom.xml

```diff
@@ -177,12 +177,6 @@
             <scope>test</scope>
         </dependency>
 
-        <dependency>
```

Comment:
Can we revert this ?

Reply:
sure but maven-dependency plugin complained about that. I guess it is not needed - all checks passed without it

Reply:
I think it is transitively  added by http-client dependency in trino-plugin-toolkit

Reply:
it must be added as this dependency is added by http-client